micro_manager/snapshot/dataset.py

```diff
@@ -0,0 +1,238 @@
+from importlib import metadata
+import os
+from datetime import datetime
+
+import numpy as np
+
+try:
+    import h5py
+except ImportError:
+    raise ImportError(
+        "The Micro Manager snapshot computation requires the h5py package."
+    )
+
+import numpy as np
+
+try:
+    import h5py
+except ImportError:
+    raise ImportError(
+        "The Micro Manager snapshot computation requires the h5py package."
+    )
```

Comment:
Why does this piece of code occur twice?

Reply:
That must be some rebasing error :grimacing: